draft/7/4d-projection-and-basic-hypercube-twists/index.md

```diff
@@ -0,0 +1,133 @@
+# Cubing Standard #7: 4D Projection and Basic Hypercube Twists (DRAFT)
+
+Status: `DRAFT`  
+Prescriptive or descriptive: descriptive (this standard attempts to describe existing conventions)
+
+This standard describes the basic moves for a NxNxNxN facet-turning hypercube.
+
+## 7.1 Polytope elements
+
+When possible, pieces are named according to the rank of the element of the polytope they correspond to. Let _n_ be the rank of the puzzle.
```

Comment:
Hmm, above talks about `N×N×N×N`? Does the capitalization provide a semantic distinction, or can they be the same?